CHANGELOG.md

```diff
@@ -4,6 +4,16 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
+### 0.13.0
+
+**Released**: 2021.05.06
+
+**Commit Delta**: [Change from 0.13.0 release](https://github.com/plus3it/tardigrade-ci/compare/0.12.4..0.13.0)
```

Comment:
The link title is for the changes from the **prior** release...

```suggestion
**Commit Delta**: [Change from 0.12.4 release](https://github.com/plus3it/tardigrade-ci/compare/0.12.4..0.13.0)
```

Reply:
Ohhh...oops.